### PR TITLE
Persist

### PR DESCRIPTION
**Pull request by tarasivanchyshyn**

### CODE REVIEW

File: src/components/Button.module.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  background: $color-primary;
+  @include border;
+  cursor: pointer;
+}
+
+.button:hover,
+.button:active,
+.button:focus {
+  @include btn-action;
+}
```
Comment (Yuliia143) [inline src/components/Button.module.scss]: Can move it inside .button

. button {
&:hover
}

File: src/components/Modal.module.scss [hunk 2]
```diff
@@ -1,75 +1,57 @@
...
 
 input::-webkit-calendar-picker-indicator {
   cursor: pointer;
 }
 
 .input:focus {
-  outline: 1px solid #858585;
-  border: none;
-  box-shadow: 0 2px 3px rgba(0, 0, 0, 0.25);
+  @include input-outline;
```
Comment (Yuliia143) [inline src/components/Modal.module.scss]: The same
.input {
&:focus {}
}

File: src/components/Modal.tsx [hunk 3]
```diff
@@ -40,7 +41,7 @@ function ModalOverlay(props: UniversalProps) {
     const enteredExpirationDate = todoExpirationtRef.current!.value;
 
     if (
-      enteredText.trim().length === 0 ||
+      !enteredText.trim().length ||
```
Comment (Yuliia143) [inline src/components/Modal.tsx]: empty string - false, try to simplify

File: src/components/SortButton.module.scss [hunk 4]
```diff
@@ -35,46 +36,44 @@
...
   color: white;
 }
 
 .dropdownItem svg {
   margin: 0 1rem 0.3rem 0;
-  color: grey;
+  color: $color-dark-grey;
 }
 
 .dropdownItem:hover svg {
```
Comment (Yuliia143) [inline src/components/SortButton.module.scss]: The same
.dropdownItem {
&:hover{},
svg {}
}

File: src/components/Modal.tsx (hunk 3 above)
Comment (tarasivanchyshyn) [reply src/components/Modal.tsx]: don't understand how can i simplify it more, with **!enteredText.trim().length** i'm checking for entered spaces

Comment (Kost927) [reply src/components/Modal.tsx]: do not use .length. coz !enteredText.trim().length  and !enteredText.trim() will return the same false coz  "" (empty string) is false and !enteredText.trim().length   will return 0 which is also false.
If you don't type any text in input (empty input) it is an empty string ("").
.length you can use for arrays, coz empty array ([]) => true, but array.length is false (if it is empty)

Comment (tarasivanchyshyn) [reply src/components/Modal.tsx]: aa, ok, thanks

**Review by Yuliia143 — APPROVED** [review]

**Review by Yuliia143 — APPROVED** [review]

**Pull request merged**